src/lib/common/versionInfo.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_LIB_COMMON_VERSIONINFO_H_

```

Comment:
I understand this file is no longer needed and should be removed... or I'm missing something?

Reply:
I had deleted this file, I probably missed it when it came back after a fork reset. 

Fixed in 957bf89